entities.go

```diff
@@ -15,12 +15,14 @@ import (
 )
 
 type Network struct {
-	id       string
-	endpoint *net.UDPAddr
-	seed     []byte
-	pubkey   wgtypes.Key
-	route    string
-	ifname   string
+	id         string
+	endpoint   *net.UDPAddr
```

Comment:
@leomos I would like to update this to a string in a future PR. Thoughts?

Reply:
it's a good idea to keep consistency with what we agreed over the `gateway{_v6}` fields.
If you can change it in this PR it's great, otherwise no problem in doing that later.

Reply:
I won't make this change in this patch because it's a slightly bigger change. Right now the daemon saves an IP address, which is not even the IP address at the moment of the tunnel creation. I think this should be an FQDN which is resolved at tunnel creation. It will still not solve all the problems the wg-quick solves, but it's a step in that direction and we can iterate over it.